lua/entities/gmod_wire_thruster.lua

```diff
@@ -114,36 +113,37 @@ function ENT:SetForce( force, mul )
 		self.force = force
 		self:ShowOutput()
 	end
-	mul = mul or 1
+	self.mul = mul or 1
+	self.needcalc = true
 
 	local phys = self:GetPhysicsObject()
 	if (!phys:IsValid()) then
 		Msg("Warning: [",self,"] Physics object isn't valid!\n")
 		return
 	end
 
-	// Get the data in worldspace
-	local ThrusterWorldPos = phys:LocalToWorld( self.ThrustOffset )
-	local ThrusterWorldForce = phys:LocalToWorldVector( self.ThrustOffset * -1 )
-
-	// Calculate the velocity
-	ThrusterWorldForce = ThrusterWorldForce * self.force * mul * 50
-	self.ForceLinear, self.ForceAngle = phys:CalculateVelocityOffset( ThrusterWorldForce, ThrusterWorldPos );
-	self.ForceLinear = phys:WorldToLocalVector( self.ForceLinear )
-
 	if self.neteffect then
-		-- self.ForceLinear is 0 if the thruster is frozen
-		self.effectforce = ThrusterWorldForce:Length()
+		self.effectforce = self.ThrustOffset:Length() * (self.force * self.mul * -50)
```

Comment:
My mistake here. Should be positive 50 instead of negative and also don't need parentheses.